docs/DeveloperGuide.md

```diff
@@ -200,6 +200,65 @@ the filtered transaction with a success message.
   - 2a1. UniCa$h displays an error message with the correct command format.
   - Use case resumes at step 1.
 
+## Implementation
+
+This section aims to describe the implementation of the features in UniCash.
+
+There are 3 main group of features we have came up with.
+1. Transaction
+2. Budget
+3. General
+
+### Transaction
+
+<img src="images/unicash/TransactionClassDiagram.png" width="700" />
+
+UniCash tracks transactions with the use of `TransactionList` and `Transaction`. `TransactionList` acts
```

Comment:
I think given our naming scheme right, we make it a point to use `UniCa$h` as much as possible when referring to the Application as a whole, and `UniCash` when referring to the model that contains transactionsList. Just to be safe about naming bugs and all that, what do you think?

Reply:
Yeah sure we could stick to that